src/erlcloud_kinesis.erl

```diff
@@ -406,6 +407,57 @@ put_record(StreamName, PartitionKey, Data, ExplicitHashKey, Ordering, Config) wh
   Json = [{<<"StreamName">>, StreamName}, {<<"PartitionKey">>, PartitionKey}, {<<"Data">>, base64:encode(Data)}, {<<"ExplicitHashKey">>, ExplicitHashKey}, {<<"SequenceNumberForOrdering">>, Ordering}],
   erlcloud_kinesis_impl:request(Config, "Kinesis_20131202.PutRecord", Json).
 
+%%------------------------------------------------------------------------------
+%% Kinesis API:
+%% [https://docs.aws.amazon.com/kinesis/latest/APIReference/API_PutRecords.html]
+%%
+%% ===Example===
+%%
+%% This operation puts a data record into an Amazon Kinesis stream from a producer.
+%%
+%% `
+%% erlcloud_kinesis:put_records(<<"test">>, [{<<"asdasd">>, <<"asdasd">>, <<"asdasd">>}]).
+%% '
+%%
+%% @end
+%%------------------------------------------------------------------------------
+
+-spec put_records(string(), Items) -> proplist() when
+      Items :: [Item],
+      Item :: {string(), string()}
+            | {string(), string(), string()}.
+
+put_records(StreamName, Items) ->
+    put_records_common(StreamName, Items, default_config()).
+
+-spec put_records(string(), Items, Config) -> proplist() when
+      Items :: [Item],
+      Item :: {string(), string()}
+            | {string(), string(), string()},
+      Config :: aws_config().
+
+put_records(StreamName, Items, Config) ->
+    put_records_common(StreamName, Items, Config).
+
+put_records_common(StreamName, Items, Config) ->
```

Comment:
Why? Why not just do this in put_records/3, and make put_records/2 call it? Seems like useless abstraction.